docs/sop.rst

```diff
@@ -1,31 +0,0 @@
-============================
-Standard Operating Procedure
```

Comment:
I would probably port a part of this section in the localization document. Eg why locales may not be updated in the site b/c of broken strings in pontoon.